includes/filters.php

```diff
@@ -117,13 +129,36 @@ function give_akismet( $spam ) {
 	$ignore = array( 'HTTP_COOKIE', 'HTTP_COOKIE2', 'PHP_AUTH_PW' );
 
 	foreach ( $_SERVER as $key => $value ) {
-		if ( ! in_array( $key, (array) $ignore ) ) {
-			$args["$key"] = $value;
+		if ( ! in_array( $key, $ignore, true ) ) {
+			$args[$key] = $value;
 		}
 	}
 
+	$response = give_akismet_spam_check_post( $args );
+	$spam     = 'true' === $response[1];
+
+	// Log spam information.
+	if( $spam ) {
+		give_record_log(
+			sprintf(
+				'Spam flagged for %1$s (%2$s)',
+				$args['comment_author'],
+				$args['comment_author_email']
+			),
+			sprintf(
+				'<p><strong>%1$s</strong><pre>%2$s</pre></p><strong>%3$s</strong><pre>%4$s</pre><p>',
+				__('Request', 'give-zapier'),
+				print_r( $args, true ),
+				__('Response', 'give-zapier'),
```

Comment:
`give-zapier` is incorrect

Reply:
zapier and give interconnected, because I was working on them, I will update it

Reply:
https://github.com/impress-org/givewp/commit/3ccf0fe590219703cc7ca7fa0809c9a84664c4c8

Reply:
I this must be fixed by an auto fixer. I will create issue to setup tools like give-campaigns